src/alerts/flash.scss

```diff
@@ -1,155 +1 @@
-  border-radius: $border-radius;
-
-  p:last-child {
-    margin-bottom: 0;
-  }
-
-  .octicon {
-    // stylelint-disable-next-line primer/spacing
-    margin-right: 12px;
-  }
-}
-
-// Contain the flash messages
-.flash-messages {
-  margin-bottom: $spacer-4;
-}
-
-// Close button
-.flash-close:not(.Banner-close) {
-  float: right;
-  padding: $spacer-3;
-  margin: -$spacer-3;
-  text-align: center;
-  cursor: pointer;
-
-  // Undo `<button>` styles
-  background: none;
-  border: 0;
-  appearance: none;
-
-  &:hover {
-    opacity: 0.7;
-  }
-
-  &:active {
-    opacity: 0.5;
-  }
-
-  .octicon {
-    margin-right: 0;
-  }
-}
-
-// Action button
-.flash-action:not(.Banner-actions) {
-  float: right;
-  // stylelint-disable-next-line primer/spacing
-  margin-top: -3px;
-  margin-left: $spacer-4;
-  background-clip: padding-box;
-
-  &.btn .octicon {
-    margin-right: $spacer-1;
-    color: var(--color-fg-muted);
-  }
-
-  &.btn-primary {
-    background-clip: border-box;
-
-    .octicon {
-      color: inherit;
-    }
-  }
-}
-
-//
-// Color variations
-//
-
-.flash:not(.Banner) {
-  color: var(--color-fg-default);
-  background-image: linear-gradient(var(--color-accent-subtle), var(--color-accent-subtle));
-  border-color: var(--color-accent-muted);
-
-  .octicon {
-    color: var(--color-accent-fg);
-  }
-}
-
-.flash-warn:not(.Banner) {
-  color: var(--color-fg-default);
-  background-image: linear-gradient(var(--color-attention-subtle), var(--color-attention-subtle));
-  border-color: var(--color-attention-muted);
-
-  .octicon {
-    color: var(--color-attention-fg);
-  }
-}
-
-.flash-error:not(.Banner) {
-  color: var(--color-fg-default);
-  background-image: linear-gradient(var(--color-danger-subtle), var(--color-danger-subtle));
-  border-color: var(--color-danger-muted);
-
-  .octicon {
-    color: var(--color-danger-fg);
-  }
-}
-
-.flash-success:not(.Banner) {
-  color: var(--color-fg-default);
-  background-image: linear-gradient(var(--color-success-subtle), var(--color-success-subtle));
-  border-color: var(--color-success-muted);
-
-  .octicon {
-    color: var(--color-success-fg);
-  }
-}
-
-//
-// Layout variations
-//
-
-.flash-full:not(.Banner) {
-  // stylelint-disable-next-line primer/spacing
-  margin-top: -1px;
-  border-width: $border-width 0;
-  border-radius: 0;
-}
-
-// A banner rendered at the top of the page.
-.flash-banner {
-  position: fixed;
-  top: 0;
-  z-index: 90;
-  width: 100%;
-  border-top: 0;
-  border-right: 0;
-  border-left: 0;
-  border-radius: 0;
-}
-
-// Makes sure the background is opaque to cover any content underneath
-.flash-full,
-.flash-banner {
-  background-color: var(--color-canvas-default);
-}
-
-// FIXME deprecate this
-.warning {
-  padding: $em-spacer-5;
-  // stylelint-disable-next-line primer/spacing
-  margin-bottom: 0.8em;
-  font-weight: $font-weight-bold;
-  background-color: var(--color-attention-subtle);
-}
+@import '@primer/view-components/app/components/primer/beta/flash';
```

Comment:
Figured out what was wrong with those failures, removing the `.css` here fixed it. We'll have to remember that it could cause problems

Reply:
💡